javaee/jobexecutor-ra/src/main/java/org/camunda/bpm/container/impl/threading/ra/inflow/JcaInflowExecuteJobsRunnable.java

```diff
@@ -73,7 +73,6 @@ protected void executeJob(String nextJobId, CommandExecutor commandExecutor) {
       } catch (ResourceException e) {
         log.log(Level.WARNING, "ResourceException while invoking beforeDelivery() on MessageEndpoint '"+endpoint+"'", e);
       }
-      JobFailureCollector jobFailureCollector = null;
       try {
         jobFailureCollector = ((JobExecutionHandler)endpoint).executeJob(nextJobId, commandExecutor);
```

Comment:
@mboskamp, just to make sure I understand correctly since I'm not sure how to test this:

Even though `ExecuteJobHelper.executeJob` is not called explicitly anymore, calling `endoint.executeJob` here ensures that the `ExecuteJobHelper` is called through an instance of `JobExecutionHandler` and the `Process Data Context for logging` feature is intact, i.e. MDC properties are still added?

Reply:
Not sure, if I understand you correctly.
`ExecuteJobHelper.executeJob` is called from the JobExecutionHandler implementation and should behave the same in this regard as the main implementation: https://github.com/camunda/camunda-bpm-platform/blob/master/javaee/ejb-service/src/main/java/org/camunda/bpm/container/impl/ejb/JobExecutionHandlerMDB.java#L44